resource/reactivedrop_brazilian.txt

```diff
@@ -3700,7 +3700,7 @@
 "RD_SPEEDRUN_TIL_ROAD_TO_DAWN_NAME"		"Speedrun Estrada para o Amanhecer"
 "[english]RD_SPEEDRUN_TIL_ARCTIC_INFILTRATION_NAME"		"Arctic Infiltration Speedrun"
 "RD_SPEEDRUN_TIL_ARCTIC_INFILTRATION_NAME"		"Speedrun Infiltração do Ártico"
-"[english]RD_SPEEDRUN_TIL_AREA9800_NAME"		"Area9800 Landingzone Speedrun"
```

Comment:
This is a Tilarus-5 map, not part of the Area9800 campaign.

Reply:
But it's close to the same map as in the Area 9800 campaign (which makes it even stranger if you remember Area 9800 should be the name of a whole planet). Just so I get it right:
* The campaign "Area 9800" has a map named "Landing Zone"
* The campaign Tilarus-5 has a map named "Area9800 Landingzone"

Reply:
Yes

Tilarus-5 (originally titled Extermination) was a compilation of single-mission campaigns